modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -271,6 +271,119 @@ definition {
 		}
 	}
 
+	@description = "LPS-171913 - Verify it's possible to add an object entry with the trigger On Payment Status Update"
+	@priority = "4"
+	test CanCreateActionAddObjectEntryWithOnPaymentStatusUpdate {
+		task ("Given: An object with a field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
```

Comment:
You can use LPS when adding a Custom Object, this way avoid inconsistencies in the CI
```
ObjectAdmin.addObjectViaAPI(
	labelName = "Custom Object 171913",
	objectName = "CustomObject171913",
	pluralLabelName = "Custom Objects 171913");
```